### DISCUSSION TITLE
Blood pressure OCR + templates contribution

### DISCUSSION THREAD

**Discussion by dasch0682** [derdilla/blood-pressure-monitor-fl#452] [Ideas]
Hello. I had started checking around that idea myself and for myself. 

The idea was to provide the feature and let the user contribute the templates, adding them in a collection of models.

I am a java programmer but have no experience yet with mobile apps... But I can help maybe.

**Comment by derdilla** [comment]
Hi, that's great. Any insights, libraries, and code you provide are highly appreciated. Note that dependencies on external services integrating some large AI model would need some more discussion.

> let the user contribute the templates

I like that idea. Templates compiled in the app are fast and easy to maintain and would avoid having to do complex image rendering. If the need for in-app editing arises that can always come in a later contribution.

> I am a java programmer but have no experience yet with mobile apps

While the app is in dart, this shouldn't be a big issue. It's probably best for the heavy lifting (differentiating some text from a photo of a display and return some object with relative position and content) of the feature to be a separate package/plugin anyways (I would like to be able support IOS (no java afaik) at _some_ point, but that's just a minor concern).

(sorry for the many brackets, in case I wasn't understandable just ask and I will try to put it differently)
